test/RPBuilder.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { IVerifyCallbackArgs, IVerifyCredentialResult } from '@sphereon/wellknown-dids-client';
```

Comment:
we have a couple of builder tests already in the `RP.request.spec.ts` please move these to that test suite

Reply:
fixed